app/boards/shields/zmk_uno/zmk_uno_split.keymap

```diff
@@ -16,9 +16,7 @@
 
 /* :REMOVE ME
```

Comment:
Not sure if this is common in other IDEs (no need to change this if not), but Visual Studio and VS Code have a shortcut to comment/uncomment a block of code. At least in DTS files, it only seems to work with `//` comments though.

e.g, if this were formatted like

```dts
// &kscan_direct { status = "okay"; };
// &kscan_matrix { status = "disabled"; };
```

then you could just select both lines and press <kbd>Ctrl+K Ctrl+U</kbd> to uncomment and <kbd>Ctrl+K Ctrl+C</kbd> to comment it out again.

Reply:
Yeah, updated and tweaked the comment above for clarity.